internal/pkg/pkg.go

```diff
@@ -169,6 +184,26 @@ func (p *Pkg) DirectSubpackages() ([]*Pkg, error) {
 	return subPkgs, nil
 }
 
+// adjustDisplayPathForSubpkg adjusts the display path of subPkg relative to the RootPkgUniquePath
+// subPkg also inherits the RootPkgUniquePath value from parent package p
+func (p *Pkg) adjustDisplayPathForSubpkg(subPkg *Pkg) error {
+	if p.rootPkgParentDirPath == "" {
+		// this means p is the root package on which command is originally invoked
```

Comment:
Document the semantic of this field being empty on the struct

Reply:
Removed this check.